src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -716,8 +716,8 @@ private Task WriteNonPersonalizedFilesToVmAsync(ConnectionInfo sshConnectionInfo
                         (GetFileContent("scripts", "docker-compose.yml"), $"{CromwellAzureRootDir}/docker-compose.yml", false),
                         (GetFileContent("scripts", "cromwellazure.service"), "/lib/systemd/system/cromwellazure.service", false),
                         (GetFileContent("scripts", "mount.blobfuse"), "/usr/sbin/mount.blobfuse", true),
-                        (GetFileContent("scripts", "mysql-init", "init-user.sql"), $"{CromwellAzureRootDir}/mysql-init/init-user.sql", false),
-                        (GetFileContent("scripts", "mysql-init", "unlock-change-log.sql"), $"{CromwellAzureRootDir}/mysql-init/unlock-change-log.sql", false)
+                        (GetFileContent("scripts", "mysql", "init-user.sql"), $"{CromwellAzureRootDir}/mysql-init/init-user.sql", false),
```

Comment:
Changed to mysql-init folder name to "mysql" since "-" in the folder name gets converted to "_".

Reply:
Do we not need one for unlock-change-log.sql file?